moveit_planners/chomp/chomp_motion_planner/include/chomp_motion_planner/chomp_planner.h

```diff
@@ -51,7 +51,7 @@ class ChompPlanner
   virtual ~ChompPlanner(){};
 
   bool solve(const planning_scene::PlanningSceneConstPtr& planning_scene, const moveit_msgs::MotionPlanRequest& req,
-             const ChompParameters& params, moveit_msgs::MotionPlanDetailedResponse& res) const;
+             ChompParameters& params, moveit_msgs::MotionPlanDetailedResponse& res) const;
```

Comment:
why are you removing const from params?

Reply:
currently resolving the `const` removal issue, will commit soon.

Reply:
I had removed it earlier as I needed to change the contents of the ChompParameters &params. Didn't realize this might not be the ideal way. currently fixing it

Reply:
changed, currently using a non constant version of ChompParameters